qml/main.qml

```diff
@@ -1,12 +1,16 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.3
 import QtQuick.Controls.Material 2.1
-import QtQuick.Dialogs 1.3
 import QtQuick.Layouts 1.3
 
+// Qt6! Use 'Qt.labs.platform' instead of 'QtQuick.Dialogs'
+//import Qt.labs.platform 1.0
```

Comment:
Hi, just a simple question, this `Qt6!` is a especial tag for the compiler/interpreter or just a comment ?
Other than that the patch looks fine, but I'll probably have the master branch for Qt6 and create a separated branch for Qt5.

Reply:
Oh, I've seen this type of comment in some other projects, and just followed their rule.
It doesn't have a special meaning but makes the future search easier (like the keywords "// TODO" or "// WARNING")